udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -20,13 +20,55 @@ async function deleteCommentData() {
   await getCommentData;
   window.location.reload();
 }
+
 /** Creates an <li> element containing text. */
 function createCommentListItem(inputText) {
   const listElement = document.createElement('li');
   listElement.innerText = inputText;
   return listElement;
 }
 
+/** Creates a map to show University. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 32.880406, lng: -117.242677}, zoom: 16});
+
+  const trexMarker = new google.maps.Marker({
+    position: {lat: 32.880406, lng: -117.242677},
+    map: map,
+    title: 'UC San Diego'
+  });
+}
+
+google.charts.load('current', {'packages': ['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Fetches skill data and uses it to create a chart. */
+function drawChart() {
+  fetch('/endorse-data')
+      .then(response => response.json())
+      .then((skillVotes) => {
+        const data = new google.visualization.DataTable();
+        data.addColumn('string', 'Skill');
+        data.addColumn('number', 'Votes');
+        Object.keys(skillVotes).forEach((skill) => {
```

Comment:
`Object.keys(skillVotes).forEach((skillVote)`

Naming the singular form after the plural form is less confusing (when they are different I wonder whether some other transformation was made).

Reply:
Done